### ISSUE TITLE
Bug Report: ConHost Caption BoundingRects dissapeared

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/terminal#2176]
# Environment

Caused by Accessibility PR: https://github.com/microsoft/terminal/commit/96496d8154113ae9e0bcbe47b4bbaf33a0d4e961

# Steps to reproduce

1. open ConHost
2. open inspect.exe
3. find caption buttons

# Expected behavior

bounding rects should appear over caption buttons

# Actual behavior

bounding rects aren't there. But it seems like you can interact with them perfectly fine otherwise.

# Other notes
I think the main culprit here is `WindowUiaProvider`. Not entirely sure what caused it though.

**Comment by zadjii-msft** [comment]
after lunch: Look into `WindowUiaProvider::get_HostRawElementProvider`. That's my only lead.

**Comment by zadjii-msft** [comment]
Okay I found nothing here. What I did find was:
* this works fine for conhostv1
* this DOESN'T work for conhostv2
* this DOESN'T work for explorer
* this DOESN'T work for the rejuv'd notepad
* this works fine for terminal's titlebar when not showing the tabs in the title bar. 

Given that this works generally poorly across the OS, I'm gonna punt this. If this did work, that's cool, sorry we broke it. I don't have the UIA expertise needed to figure out how this regressed.
